### DISCUSSION TITLE
[ASK] Support Android 5 (API 22)

### DISCUSSION THREAD

**Discussion by atmanegara** [HaylLtd/cordova-plugin-background-geolocation#88] [Q&A]
I have several clients using Android version 5.1.1 (API 22), does this plugin support?

**Comment by HarelM** [comment]
You might need older version of this plugin and older version of cordova/cordova-android, but in theory it should be possible.
Note that Google play store does not allow publishing apps targeting this version...

**Reply by atmanegara** [reply]
thank you sir i will try again old version cordova For Android version 5, I didn't upload it to the Playstore because there were only a few clients, it was enough to share normally

**Reply by ivinantony** [reply]
does this older version will support android 12 too?
